client/scripts/build-tests.js

```diff
@@ -1,49 +1,167 @@
 const path = require("path")
-const esbuild = require("esbuild")
+const fs = require("fs-extra")
+
+// make sure we're at root
+process.chdir(path.resolve(__dirname, "../"))
```

Comment:
Interesting - does this still work on Windows?

Reply:
Node's `path` library is sane and hides all the Windows shenanigans. Specifically, `path.resolve` tends to convert between `/` and `\\` paths on the fly as needed.

Reply:
Also I'm on windows so of course it works on windows lmao